File: ticketProject4-DistributedJava/TicketSystem/TicketNetworking/src/main/java/network/objectprotocol/BuyTicketResponse.java
package network.objectprotocol;

import network.dto.ArtisticRepresentationDTO;

public class BuyTicketResponse implements Response {
    private ArtisticRepresentationDTO ar;

    public BuyTicketResponse(ArtisticRepresentationDTO ar) { this.ar = ar; }

    public ArtisticRepresentationDTO getAr() {
        return ar;
    }
}

File: ticketProject7-CrossPlatform/TS-crossp/networking/response/BuyTicketResponse.cs
using System;
namespace networking.protocol
{

	using ArtisticRepresentationDTO = networking.dto.ArtisticRepresentationDTO;

	[Serializable]
	public class BuyTicketResponse : Response
	{
		private ArtisticRepresentationDTO artisticRepresentationDTO;

		public BuyTicketResponse(ArtisticRepresentationDTO ar)
		{
			this.artisticRepresentationDTO = ar;
		}

		public virtual ArtisticRepresentationDTO ArtisticRepresentation
		{
			get
			{
				return artisticRepresentationDTO;
			}
		}
	}

}